### PR TITLE
update soos dash action commit hash / sarif action version / logo

### PR DESCRIPTION
**Pull request by SOOS-GSteen**
update soos dash action commit hash / sarif action version / logo

### CODE REVIEW

File: code-scanning/soos-dast-scan.yml [hunk 1]
```diff
@@ -45,6 +45,6 @@ jobs:
           target_url: "https://www.example.com/"
           output_format: "sarif"
       - name: Upload SOOS DAST SARIF Report
-        uses: github/codeql-action/upload-sarif@v2
+        uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: results.sarif
```
Comment (alexisabril) [inline code-scanning/soos-dast-scan.yml:50]:
```suggestion
          sarif_file: results.sarif

```

**Review by alexisabril — CHANGES_REQUESTED** [review]
Thank you @SOOS-GSteen for the PR! Apologies for the slow response and I've also added a newline at the end of this workflow. Would you update this PR with the latest `main`? This should otherwise be good to merge.

**Comment by SOOS-GSteen** [comment]
> Thank you @SOOS-GSteen for the PR! Apologies for the slow response and I've also added a newline at the end of this workflow. Would you update this PR with the latest `main`? This should otherwise be good to merge.

All set @alexisabril - thanks again!

**Review by alexisabril — APPROVED** [review]

**Comment by 965022-oss** [comment]
Merged CodeQL into action/workflow
from SHA f4dbb8424c

**Comment by 965022-oss** [comment]
https://github.com/actions/starter-workflows/pull/2317#issue-2159481547/Merge #2317 Branch: main 965022-oss-patch1/SHA 3a95482282, label: update code scanning/soos-dast-scan.yml,

**Review by Yedfree — APPROVED** [review]

**Pull request merged**